### PR TITLE
Add/import color flow

### LINKED ISSUE #61439: Build dummy screens for Light Importer UI
Based on these designs: TAK5FpwTvfbENJDfHc7Aq1-fi-3977%3A2909

### PR DESCRIPTION
**Pull request by bogiii**
#### Changes proposed in this Pull Request

This change brings static pages of the light importers. Since is still under the construction, the feature flag is there and it's turned on for the `development` and `wpcalypso` environments.

For more details, visit: **pcmemI-oN-p2#comment-556**

#### Testing instructions

- Capture: `/start/import-light/static`
- Scanning: `start/import-light/static/scanning`
- Colors: `start/import-light/static/colors`
- Summary: `start/import-light/static/summary`

#### Screenshots
Screenshot 2022-02-28 at 10 11 48

Screenshot 2022-03-01 at 10 09 33

Screenshot 2022-03-03 at 14 36 30

Screenshot 2022-03-01 at 14 07 24

Closes #61439

### CODE REVIEW

File: client/signup/icons/index.tsx [hunk 1]
```diff
@@ -109,6 +109,38 @@ export const tip: ReactElement = (
...
+			stroke-width="1.68"
+			stroke-linecap="round"
+			stroke-linejoin="round"
+		/>
+		<path
+			d="M28 45.4943L37.8102 54.0742L56.5989 28.4102"
+			stroke="#003A5F"
+			stroke-width="3"
+			stroke-linecap="round"
+			stroke-linejoin="round"
```
Comment (zaerl) [inline client/signup/icons/index.tsx:139]: A small change that I think needs to be made: SVG attributes must be camel case even if they are an SVG attribute (`stroke-width` -> `strokeWidth`). Same thing apply to `onboarding-3.svg`.

File: client/signup/steps/import/util.ts [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import { ImporterPlatform } from './types';
 
+export const CAPTURE_URL_RGX = /^(http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/)?[a-z0-9]+([-.][a-z0-9]+)*\.[a-z]{2,5}(:[0-9]{1,5})?(\/.*)?$/i;
```
Comment (zaerl) [inline client/signup/steps/import/util.ts:3]: I've seen that in other parts of the code [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL) is used to check if a URL is valid (not many though). So maybe in a future minor refactoring, we can substitute this regex.

**Review by zaerl — APPROVED** [review]
I wrote a couple of comments; those are not important things. Change it if you want, otherwise, you can publish this as it is. Great work.

File: client/signup/icons/index.tsx (hunk 1 above)
Comment (bogiii) [reply client/signup/icons/index.tsx:139]: You are right. I'll fix it.

File: client/signup/steps/import/util.ts (hunk 2 above)
Comment (bogiii) [reply client/signup/steps/import/util.ts:3]: Agree. I'll update it in the subsequent iterations.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7180529

Thank you @bogiii for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7180529).

**Pull request merged**